packages/pnpm-sync-lib/src/interfaces.ts

```diff
@@ -16,6 +16,13 @@ export interface ITargetFolder {
   folderPath: string;
 }
 
+export interface IFileStat {
```

Comment:
`IFileStat` is not the most intuitive name for this object, since it seems that it also tracks directories (not just files). And since `stat` is not the purpose of this data structure, but more like an implementation detail of how we create it.

Consider a different name like `ISyncItem` or `ICopyItem` etc.